libvast/CMakeLists.txt

```diff
@@ -340,10 +340,10 @@ add_library(vast::native_plugins ALIAS libvast_native_plugins)
 
 # Configure precompiled headers, except for ARM macOS because of this CMake bug:
 # https://gitlab.kitware.com/cmake/cmake/-/issues/21825
-if (NOT (CMAKE_SYSTEM_NAME MATCHES "Darwin" AND CMAKE_SYSTEM_PROCESSOR MATCHES
-                                                "arm"))
-  target_precompile_headers(libvast PUBLIC [["vast/fwd.hpp"]] <caf/fwd.hpp>)
-endif ()
+#if (NOT (CMAKE_SYSTEM_NAME MATCHES "Darwin" AND CMAKE_SYSTEM_PROCESSOR MATCHES
```

Comment:
We could also make this a bit smarter by detecting whether the CXX_COMPILER_LAUNCHER target property is set on libvast.

Reply:
That's brittle, e.g. on Debian the standard way of using ccache is to install it as a symlink for the compiler binary itself, so it can be used even without cmake support:

```
$ which cc
/usr/lib/ccache/cc
```

If anything, I'd suggest making it an option `VAST_ENABLE_PCH`, but I'm not sure who would be the target audience to enable that option.

Reply:
Then I'd prefer removing the lines altogether.